src/sr/2018-01/03/01.md

```diff
@@ -1,7 +1,14 @@
 ---
-title:  Lekcija
+title:  Bog ili bogatstvo
 date:   13/01/2018
 ---
 
-### <center>Radimo na ovoj lekciji.</center>
-<center>Molim vas, vratite se kasnije</center>
\ No newline at end of file
+### Tekstovi za proučavanje u toku sedmice: Psalam 33,6-9; Matej 19,16-22; 1. Petrova 1,18; Jevrejima 2,14.15; 2. Mojsijeva 9,14; Psalam 50,10.
```

Comment:
@AlmaBosnic It would be better to put verses on a separate line, since the "Tekstovi.." is a header.

Also is `Psalam` a synonym of Psalm? Because I did not add this one to the Bible parser.